app/views/favourites/favourite_quotes.html.erb

```diff
@@ -0,0 +1,5 @@
+<% @user_favourites.each do |quote| %>
+  <div class="well">
+    <%=  render "quotes/quote", quote: quote %>
```

Comment:
Right that makes sense.  I guess it's just partials that aren't being rendered off of an object that you need to explicit with their path if you aren't in the same folder they're defined in